README.md

```diff
@@ -27,23 +29,225 @@ You can use the package as follows:
 ```python
 from santoku.slack import SlackBotHandler
 
-slack_bot = SlackBotHandler.from_aws_secrets_manager(...)
+slack_bot = SlackBotHandler.from_aws_secrets_manager("your_secret")
+```
+
+## Content
+
+The package `santoku` contains several subpackages: `aws`, `google`, `salesforce` and `slack`. Each subpackage provides connection to different external services and are formed by a collection of modules, where each module consists of handlers for more specific services. Each handler class has unit tests to ensure the correct behaviour of the methods of these classes.
+
+### AWS
+
+AWS (Amazon Web Services) is a cloud computing platform that provides provide a set of primitive abstract technical infrastructure and distributed computing building blocks and tools.
+
+The connection to AWS has been done through [boto3](https://github.com/boto/boto3). We provide wrappers of the `boto3` SDK to make easy the operations to interact with different services.
+
+The use of this subpackage requires having AWS credentials somewhere. We provide flexibility to either keep credentials in AWS credentials/configuration file, set environment variables, or to pass them directly as arguments in the initializer of each handler class. More info on AWS configurations and credentials [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html)
+
+The unit tests in this subpackage implement mocks to the AWS services and do not pretended to access or modify the environment of your real account.
+
+#### Amazon S3
+
+Amazon Simple Storage Service (Amazon S3) is an object storage service that offers scalability, data availability, security, and performance.
+
+We provide methods to easily list and delete objects inside buckets; read and write content within S3 objects; upload a dataframe into csv or parket format to a specific location; generate and upload an Amazon Quicksight manifest in S3 in order to create analysis in Amazon Quicksight, and so on.
+
+##### How to Upload an Object to S3
+
+```python
+from aws.s3_handler import S3Handler
+
+s3_handler = S3Handler()
+s3_handler.put_object(bucket="your_bucket_name", object_key="your_object_key", content="Your object content.")
+```
+
+#### AWS Secrets Manager
+
+AWS Secrets Manager protects secrets needed to access applications, services, and IT resources. The service allows rotating, managing, and retrieving credentials, keys, and other secrets.
+
+We provide methods to get the content of a previously created secret.
+
+##### Example of usage
+
+```python
+from aws.secrets_manager_handler import SecretsManagerHandler
+
+secrets_manager = SecretsManagerHandler()
+secret_content = secrets_manager.get_secret_value(secret_name="your_secret_name")
+```
+
+This module is also used in other subpackages to simplify the instantiation: the necessary credentials to initialize certain service can be retrieved from AWS Secrets Manager.

```

Comment:
I'd set the example for alternative class methods for the rest of the modules here, so we avoid repeating ourselves later on:
"We use this service as our default credential manager. Most classes that require some form of authentication in santoku are provided with alternative class methods that retrieve the credentials directly from Secrets Manager. For example, instead of directly providing credentials to the BigQuery handling class, we simply provide it with the name of the secret where they are stored:"
```python
from santoku.google.bigquery import BigQueryHandler

bigquery = BigQueryHandler.from_aws_secrets_manager(secret_name="bq_credentials_secret_name")
```